calling-apis/post-requests-with-axios.md

```diff
@@ -0,0 +1,221 @@
+# POST Requests With axios
+
+## Introduction
+
+Matteo is making the front-end for the **Ada Trip Reserving Service™**. This service:
+
+1. helps tour guides create Trip packages catered to global travelers
+2. helps travelers find Trip packages that fit their needs, and make a reservation for that trip
+
+Matteo's front-end should make calls to [the back-end Ada Trip Reserving Service™ Trip API™™™](https://github.com/AdaGold/trip-api). The first feature he needs to build is enabling users to create new trip packages.
+
+Matteo's first step is to [research the API documentation](https://github.com/AdaGold/trip-api). He'll use the documentation and Postman to answer the following questions:
+
+1. What is the API call that lists all existing Trip packages?
+2. How many existing trips are there?
+3. What is the API call that creates a new Trip package?
+4. What is the verb and URL of that request?
+5. What are the required query parameters? What are the optional query parameters?
+
+<br/>
+
+<details style="max-width: 700px; margin: auto;">
```

Comment:
it seems like with the recent update to the Learn stylesheet, the width and margin settings are no longer needed.
```suggestion
<details>
```